pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "titan-model"
-version = "2.7.0"
+version = "2.8.0"
```

Comment:
is a feature update a patch or a minor? I know we've discussed this approximately 10 times and I can never remember